src/encoded/search_views.py

```diff
@@ -218,6 +219,41 @@ def matrix(context, request):
     return fr.render()
 
 
+@view_config(route_name='chip-seq-matrix', request_method='GET', permission='search')
+def chip_seq_matrix(context, request):
+    fr = FieldedResponse(
+        _meta={
+            'params_parser': ParamsParser(request)
+        },
+        response_fields=[
+            TitleResponseField(
+                title='ChIP Seq Matrix'
+            ),
+            TypeResponseField(
+                at_type=['ChIPSeqMatrix']
```

Comment:
Probably `ChipSeqMatrix` to match up with `HistoneChipSeqQualityMetric` @type format of this: https://www.encodeproject.org/histone-chipseq-quality-metrics/ecadc412-f9b5-42f2-8b5e-3a171f019698/?format=json

Reply:
Done